pkgs/applications/graphics/vimiv-qt/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv,
+  lib,
+  fetchFromGitHub,
+  python39Packages,
+  qt5 }:
+
+with python39Packages;
+buildPythonApplication rec {
+  pname = "vimiv-qt";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "karlch";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1pj3gak7nxkw9r9m71zsfvcaq8dk9crbk5rz4n7pravxkl5hs2bg";
+  };
+
+  nativeBuildInputs = [ qt5.wrapQtAppsHook setuptools ];
+
+  propagatedBuildInputs = [ pyqt5 py3exiv2 qt5.qtsvg ];
```

Comment:
```suggestion
  nativeBuildInputs = with python39Packages; [ wrapQtAppsHook setuptools ];

  propagatedBuildInputs = with python39Packages; [ pyqt5 py3exiv2 qtsvg ];
```

Reply:
Actually, this change breaks the package. `wrapQtAppsHook` and `qtsvg` are part of `qt5`. Should I revert your suggested removal of the `qt5` prefixes?

Reply:
Updated the suggestion to not remove the prefix.